### PR TITLE
Changes default skin tone from 35 to 110

### PR DESCRIPTION
**Pull request by listerla**
Why? It's the center ground between 1 and 220.

### CODE REVIEW

**Comment by Mechoid** [comment]
Changes aren't actually contentious, but changing the default skin-tone doesn't touch any existing characters, and is a tweak most people set anyway.

File: code/modules/client/preference_setup/general/03_body.dm [hunk 1]
```diff
@@ -449,7 +449,7 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 			pref.r_hair = 0//hex2num(copytext(new_hair, 2, 4))
 			pref.g_hair = 0//hex2num(copytext(new_hair, 4, 6))
 			pref.b_hair = 0//hex2num(copytext(new_hair, 6, 8))
-			pref.s_tone = 0
+			pref.s_tone = -75
```
Comment (Mechoid) [inline code/modules/client/preference_setup/general/03_body.dm:452]: Have you tested that this doesn't mess with non-human colors?

Comment (listerla) [reply code/modules/client/preference_setup/general/03_body.dm:452]: Played fine with my Skrell.

Comment (Mechoid) [reply code/modules/client/preference_setup/general/03_body.dm:452]: Even on a new slot changing from human to skrell etc?

Comment (listerla) [reply code/modules/client/preference_setup/general/03_body.dm:452]: Correct. I'll test it with other species some time today.

Comment (Mechoid) [reply code/modules/client/preference_setup/general/03_body.dm:452]: Other species use the same system, tis fine if it worked on the Squid

**Pull request merged**